torch/nn/modules/module.py

```diff
@@ -155,7 +157,7 @@ def register_module_forward_hook(hook: Callable[..., None]) -> RemovableHandle:
 
 
 def register_module_backward_hook(
-    hook: Callable[['Module', _grad_t, _grad_t], Union[None, Tensor]]
+    hook: Callable[['Module', _grad_t, _grad_t], Union[None, _grad_t]]
```

Comment:
I think the signature here was wrong.

Note: This change is not related to `backward_pre_hook`